### PR TITLE
Add propagator config to README

### LINKED ISSUE #822: Add documentation for propagator config parameter
#425 alludes to the difficulty users have in figuring out how to configure an application to use a specific propagator (e.g. b3). 

This feature is not documented like most other [config parameters](https://github.com/open-telemetry/opentelemetry-java-instrumentation#configuration-parameters-subject-to-change) in the main README of the project.

I think it would be valuable to explain, in the main project README, how the `ota.propagators` parameter works.

I have found the diagrams [here](https://github.com/openzipkin/b3-propagation) to be helpful in understanding header propagation models for the SignalFx Java Instrumentation agent - something like that could also be referenced if these docs changes are made.

### PR DESCRIPTION
**Pull request by trask**
Closes #822

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ attributes to stdout. It is used mainly for testing and debugging.
 | ota.exporter=logging        | OTA_EXPORTER=logging        | To select logging exporter                                                   |
 | ota.exporter.logging.prefix | OTA_EXPORTER_LOGGING_PREFIX | An optional string that is printed in front of the span name and attributes. |
 
+#### Propagator
+
+The propagator controls which distributed tracing header format is used.
+
+| System property | Environment variable | Purpose                                                                                 |
+|-----------------|----------------------|-----------------------------------------------------------------------------------------|
+| ota.propagators | OTA_PROPAGATORS      | Default is "tracecontext" (W3C). Other supported values are "b3", "b3single", "jaeger". |
```
Comment (anuraaga) [inline README.md:110]: Should document this is a comma separated list (or at least I think it is)

Comment (anuraaga) [reply README.md:110]: Oops found #833

**Comment by iNikem** [comment]
What do you think about feature creep and changing the prefix from `ota` to `otel`?

**Comment by trask** [comment]
> What do you think about feature creep

I think this config is necessary to support zipkin and jaeger users.

> and changing the prefix from `ota` to `otel`?

👍 #675

**Comment by iNikem** [comment]
I mean while you are at documenting it, you can change it to desired final state `otel.propagator` :)

**Comment by jamesylgan** [comment]
As a side note, this will be a great spot for #637 documentation

**Pull request merged**